MIGRATION.md

```diff
@@ -3,6 +3,7 @@
 This guide provides detailed instructions for migrating your project from one version of the `@metamask/design-tokens` to another.
 
 - [From version 2.1.1 to 3.0.0](#from-version-211-to-300)
+- [From version 3.0.0 to 4.0.0](#from-version-300-to-400)
```

Comment:
this wouldn't be a `4.0`, this would be part of `3.0` right? @georgewrmarshall 
```suggestion
- [From version 3.0.0 to 4.0.0](#from-version-300-to-400)
```
```suggestion
```

Reply:
nevermind i see that v3 already got released